### PR TITLE
Add GitHub teams for cosi-driver-minio

### PR DESCRIPTION
**Pull request by nikhita**
Ref: https://github.com/kubernetes/org/issues/2631

/assign @ameukam

### CODE REVIEW

File: config/kubernetes-sigs/sig-storage/teams.yaml [hunk 1]
```diff
@@ -94,6 +94,24 @@ teams:
...
     privacy: closed
+  cosi-driver-minio-admins:
+    description: Admin access to cosi-driver-minio
+    members:
+    - jsafrane
+    - msau42
+    - saad-ali
+    - xing-yang
+    privacy: closed
+  cosi-driver-minioc-maintainers:
```
Comment (xing-yang) [inline config/kubernetes-sigs/sig-storage/teams.yaml:105]: This says "cosi-driver-minioc-maintainers".  Should it be "cosi-driver-minio-maintainers"?

Comment (ameukam) [reply config/kubernetes-sigs/sig-storage/teams.yaml:105]: cc @nikhita

Comment (nikhita) [reply config/kubernetes-sigs/sig-storage/teams.yaml:105]: > This says "cosi-driver-minioc-maintainers". Should it be "cosi-driver-minio-maintainers"?


oops, yes. created https://github.com/kubernetes/org/pull/2654 to fix this

**Pull request merged**